.github/workflows/benchmark.yml

```diff
@@ -141,7 +141,7 @@ jobs:
           SPARK_HOME=`pwd` SPARK_TPCDS_DATA=`pwd`/tpcds-sf-1 ./mvnw -B -Prelease -Dsuites=org.apache.spark.sql.CometTPCDSQuerySuite test
         env:
           SPARK_TPCDS_JOIN_CONF: |
-            spark.sql.autoBroadcastJoinThreshold=-1
+            spark.sql.autoBroadcastJoinThreshold=-2 # Set to -2 to force Spark to sort the results
```

Comment:
Spark `TPCDSQueryTestSuite` checks if `spark.sql.autoBroadcastJoinThreshold` is -1 to decide whether to sort returned results.